backend/compact-connect/lambdas/license-data/tests/resources/dynamo/license.json

```diff
@@ -1,27 +1,28 @@
 {
-  "given_name": "string",
-  "middle_name": "string",
-  "family_name": "string",
-  "license_type": "speech language",
-  "date_of_issuance": "2024-06-06",
-  "date_of_renewal": "2024-06-06",
-  "date_of_expiration": "2024-06-06",
+  "providerId": "89a6377e-c3a5-40e5-bca5-317ec854c570",
+  "givenName": "Björk",
+  "middleName": "Gunnar",
+  "familyName": "Guðmundsdóttir",
+  "licenseType": "speech-language pathologist",
+  "dateOfIssuance": "2024-06-06",
+  "dateOfRenewal": "2024-06-06",
+  "dateOfExpiration": "2024-06-06",
   "status": "active",
   "ssn": "123-12-1234",
   "npi": "0608337260",
-  "home_state_street_1": "string",
-  "home_state_street_2": "string",
-  "home_state_city": "string",
-  "home_state_postal_code": "string",
-  "date_of_birth": "2024-06-06",
-  "pk": "123-12-1234",
+  "homeStateStreet1": "123 A St.",
+  "homeStateStreet2": "Apt 321",
+  "homeStateCity": "Birmingham",
+  "homeStatePostalCode": "35004",
+  "dateOfBirth": "2024-06-06",
+  "pk": "89a6377e-c3a5-40e5-bca5-317ec854c570",
   "sk": "aslp/co/license-home",
-  "compact_jur": "aslp/co",
-  "date_of_update": "2024-07-01",
+  "licenseHomeProviderId": "89a6377e-c3a5-40e5-bca5-317ec854c570",
+  "compactJur": "aslp/co",
+  "dateOfUpdate": "2024-07-01",
   "type": "license-home",
   "compact": "aslp",
   "jurisdiction": "co",
-  "upd_ssn": "2024-07-01/123-12-1234",
-  "fam_giv_mid_ssn": "string/string/string/123-12-1234",
-  "birth_month_day": "06-06"
+  "famGivMid": "Gu%C3%B0mundsd%C3%B3ttir/Bj%C3%B6rk/Gunnar",
```

Comment:
It has to do with the function of the index field. Queries on the db can be things like '`famGivMid` starts with `{family_name}/{given_name}/`'. Because a whole host of characters are allowed in name fields, including `/`, it's possible to have some unexpected behavior happen for queries involving names with unusual characters (kind of similar to how SQL injection works). We could take an approach of specifically handling the `/` character in names, but we'd have to decide how to escape it, then handle that particular escape sequence appearing in a name, etc. Rather than rolling our own custom escaping paradigm, we're just eliminating the possibility of unexpected behavior resulting from those sequences by url-encoding the fields before replicating them into the index field.

Reply:
Got it, thanks for the explanation!